radio/src/gui/480x272/radio_hardware.cpp

```diff
@@ -169,7 +170,7 @@ bool menuRadioHardware(event_t event)
         break;
       }
 
-#if defined (PCBX10)
+#if defined (PCBX10) || defined (PCBX12S)
```

Comment:
You'd want to simply remove this define since this file is only used for X10 and X12S...